windows/win_acl.ps1

```diff
@@ -0,0 +1,165 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2015, Phil Schwartz <schwartzmx@gmail.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+# win_acl module (File/Resources Permission Additions/Removal)
+$params = Parse-Args $args;
+
+$result = New-Object psobject @{
+    win_acl = New-Object psobject
+    changed = $false
+}
+
+If ($params.src) {
+    $src = $params.src.toString()
+
+    If (-Not (Test-Path -Path $src -PathType Leaf -Or Test-Path -Path $src -PathType Container)) {
```

Comment:
This line actually fails when I run the compiled script on my node. Are you simply testing whether the path exists - either in form of a file or container? In that case you can just do ```if (test-path -path $src) {```